src/unistd.rs

```diff
@@ -478,9 +491,8 @@ fn dup3_polyfill(oldfd: RawFd, newfd: RawFd, flags: OFlag) -> Result<RawFd> {
 /// pages for additional details on possible failure cases.
 #[inline]
 pub fn chdir<P: ?Sized + NixPath>(path: &P) -> Result<()> {
-    let res = path.with_nix_path(|cstr| {
-        unsafe { libc::chdir(cstr.as_ptr()) }
-    })?;
+    let res =
```

Comment:
This looks like purely a formatting change.  Could you please separate formatting changes into a separate commit?

Reply:
Sounds good, done: https://github.com/nix-rust/nix/pull/1908. Some background: to edit this code, I would temporarily remove the `feature!` macro wrappers which meant everything got reformatted.

Reply:
Are you using some kind of fancy editor, yet one that doesn't understand feature flags?

Reply:
Nah, this one is due to https://github.com/rust-lang/rustfmt/issues/3253. Which actually, I wonder if we can just move the `#![$meta:meta]` into the macro call so that formatting will finally work properly. Lemme try that.

Reply:
Tried it, didn't work. :(